### PR TITLE
Marketing stories

### PR DESCRIPTION
**Pull request by Iamsamwhoami**
This PR adds new marketing screens 🔥 

I implemented this feature with Flow, Form, and Presentation, three quite nice libraries from iZettle, I think the code ended up looking super neat 👌 

The feature isn't really done, but to avoid this getting way to big I am PRing part of it now 😄

### CODE REVIEW

**Comment by palmfjord** [comment]
> but to avoid this getting way to big

screenshot 2018-11-29 at 13 55 21

😂😭

**Comment by Iamsamwhoami** [comment]
@palmfjord 

to be fair

screenshot 2018-11-29 at 13 59 55

😅

**Comment by palmfjord** [comment]
@Iamsamwhoami noted 😂

File: Hedvig/Data/API.swift [hunk 1]
```diff
@@ -651,6 +651,136 @@ public final class CurrentChatResponseSubscription: GraphQLSubscription {
   }
 }
 
+public final class MarketingStoriesQuery: GraphQLQuery {
+  public let operationDefinition =
+    "query MarketingStories {\n  marketingStories(orderBy: importance_ASC) {\n    __typename\n    asset {\n      __typename\n      mimeType\n      url\n    }\n    duration\n  }\n}"
```
[the remark this answers is not in the document]
Comment (Iamsamwhoami) [reply Hedvig/Data/API.swift]: @palmfjord the whole `Hedvig/Data/API.swift` is generated by Apollo 😂

[the remark this answers is not in the document]
Comment (Iamsamwhoami) [reply Hedvig/Data/API.swift]: And apparently there is no easy way to remove it from source control due to it being needed during build and the script that creates it requires itself to be ran from xcode 🙃

[the remark this answers is not in the document]
Comment (palmfjord) [reply Hedvig/Data/API.swift]: Good 😂😂

**Pull request merged**